### PR TITLE
Add AKS prod config for sscs cor backend

### PR DESCRIPTION
**Pull request by jackmaloney**

### CODE REVIEW

File: k8s/prod/common/sscs/sscs-cor-backend.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "d6c3dd58-af16-425d-8c4a-f491e895e35d"
+      image: hmctspublic.azurecr.io/sscs-cor/backend:prod-3bc6785f
+      environment:
+        IDAM_URL: https://idam-api.platform.hmcts.net
+        PDF_SERVICE_BASE_URL: https://docmosis.platform.hmcts.net/rs/render
+        PDF_SERVICE_HEALTH_URL: https://docmosis.platform.hmcts.net/rs/status
+        PDF_SERVICE_CONVERT_URL: https://docmosis.platform.hmcts.net/rs/convert
+        IDAM_OAUTH2_REDIRECT_URL: https://sscs-case-loader-prod.service.core-compute-prod.internal
+        IDAM_SSCS_URL: https://sscs-case-loader-prod.service.core-compute-prod.internal
```
Comment (adusumillipraveen) [inline k8s/prod/common/sscs/sscs-cor-backend.yaml]: is this right ?

Comment (jackmaloney) [reply k8s/prod/common/sscs/sscs-cor-backend.yaml]: Yeh

Comment (adusumillipraveen) [reply k8s/prod/common/sscs/sscs-cor-backend.yaml]: case loader being a job shouldn't have a end point accessible ? Is this some kind of workaround ?

Comment (timja) [reply k8s/prod/common/sscs/sscs-cor-backend.yaml]: it's whitelisted in IDAM as a 'redirect_uri' it's only used to get an authentication token

**Pull request merged**